commitlint.config.ts

```diff
@@ -151,7 +151,7 @@ module.exports = {
                         header: any;
                         body: any;
                     },
-                    when: string
+                    when: When
```

Comment:
this change is wrong, the When type lives in our code, not in commitlint code

Reply:
fixed it